scripts/aws_ecs_benchmarks_cdn.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# make sure the following line is added to `~/.bashrc`
+source "$HOME/.cargo/env"
+
+# assign local ip
+ip=`curl http://169.254.169.254/latest/meta-data/local-ipv4`
+orchestrator_url=http://"$ip":4444
+cdn_marshal_address="$ip":9000
+keydb_address=redis://"$ip":6379
+
+# build to get the bin in advance, uncomment the following if built first time
+# just async_std example validator-push-cdn -- http://localhost:4444 &
+# # remember to sleep enough time if it's built first time
+# sleep 5m
+# for pid in $(ps -ef | grep "validator-push-cdn" | awk '{print $2}'); do sudo kill -9 $pid; done
+
+# docker build and push
+docker build . -f ./docker/validator-cdn.Dockerfile -t ghcr.io/espressosystems/hotshot/validator-webserver:main-async-std
+docker push ghcr.io/espressosystems/hotshot/validator-webserver:main-async-std
+
+# ecs deploy
+ecs deploy --region us-east-2 hotshot hotshot_centralized -i centralized ghcr.io/espressosystems/hotshot/validator-webserver:main-async-std
+ecs deploy --region us-east-2 hotshot hotshot_centralized -c centralized ${orchestrator_url} # http://172.31.8.82:4444
+
+# runstart keydb
+# docker run --rm -p 0.0.0.0:6379:6379 eqalpha/keydb &
+# server1: broker and marshal
+just async_std example cdn-marshal -- -d redis://localhost:6379 -b 9000 &
+# remember to sleep enough time if it's built first time
+just async_std example cdn-broker -- -d redis://localhost:6379 \
+    --public-bind-endpoint 0.0.0.0:1740 \
+    --public-advertise-endpoint local_ip:1740 \
+    --private-bind-endpoint 0.0.0.0:1741 \
+    --private-advertise-endpoint local_ip:1741 &
+# server2: broker
+# just async_std example cdn-broker -- -d ${keydb_address} \
+#     --public-bind-endpoint 0.0.0.0:1740 \
+#     --public-advertise-endpoint local_ip:1740 \
+#     --private-bind-endpoint 0.0.0.0:1741 \
+#     --private-advertise-endpoint local_ip:1741 &
+
+# for a single run
+# total_nodes, da_committee_size, transactions_per_round, transaction_size = 100, 10, 1, 4096
+# for iteration of assignment
+# see `aws_ecs_nginx_benchmarks.sh` for an example
+for total_nodes in 10 # 50 100
+do
+    for da_committee_size in 5 #10 50 100
+    do
```

Comment:
Just curious - how many iterations does this run?

Reply:
I haven't run this for large parameters so I just comment out them, and I'll update this script before each run, but for the script merged in main maybe I should get it a complete version.